qemu/tests/hotplug_mem.py

```diff
@@ -0,0 +1,116 @@
+import time, logging
+from autotest.client.shared import error, utils
+from virttest import utils_misc, utils_test, env_process
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+
+class MemoryHotplugSimple(MemoryHotplugTest):
+
+      @error.context_aware
+      def verify_memory(self, vm):
+          error.context("Verify memory info", logging.info)
+          timeout = float(self.params.get("wait_resume_timeout", 30))
+          # Notes:
+          #    some sub test will pause VM, here need to wait VM resume
+          # then check memory info in guest.
+          utils_misc.wait_for(lambda: not vm.is_paused, timeout=timeout)
+          vm.verify_alive()
+          guest_mem_size = self.get_guest_mem(vm)
+          vm_mem_size = self.get_vm_mem(vm)
+          threshold = vm_mem_size * 0.06
+          if abs(guest_mem_size - vm_mem_size) > threshold:
+              msg = "Assigned %.2f M memory to %s" % (vm_mem_size, vm.name)
+              msg += ", but only %.2f M memory detect in OS" % guest_mem_size
+              raise error.TestFail(msg)
+          utils_misc.verify_host_dmesg()
+
+      @error.context_aware
+      def verify_qemu_rss(self, vm):
+          error.context("Check qemu process RSS", logging.info)
+          qemu_rss = self.get_qemu_rss(vm)
+          guest_mem = self.get_vm_mem(vm)
+          max_rss = guest_mem * 1.02 + 200
+          if qemu_rss > max_rss:
+              raise error.TestFail("Qemu RSS(%.2fMB) " % qemu_rss +
+                                   "bigger that expect(%.2fMB)" % max_rss)
+
+      @error.context_aware
+      def run_sub_test(self, sub_type, sub_name, tag=None, parallel=False):
+          args = (self.test, self.params, self.env, sub_type, tag)
+          func = utils_test.run_virt_sub_test
+          info = "Run sub test %s" % sub_name
+          if parallel:
+              info += " in background"
+          error.context(info, logging.info)
+          if parallel:
+              sub_test = utils.InterruptedThread(func, args)
```

Comment:
line 42-45
Why not writting as below: 
+          if parallel:
+              info += " in background"
+              error.context(info, logging.info)
+              sub_test = utils.InterruptedThread(func, args)